server/src/entities/plugin.entity.ts

```diff
@@ -6,9 +6,11 @@ import {
   OneToOne,
   PrimaryGeneratedColumn,
   UpdateDateColumn,
+  Unique
 } from 'typeorm';
 import { File } from 'src/entities/file.entity';
 
+@Unique(['pluginId'])
```

Comment:
We cannot make use of this as it works with TypeORM synchronisation feature. We write custom migrations and will have to create one for this.

Check the /migrations directory